internal/observability.go

```diff
@@ -28,80 +28,86 @@ import (
+	tagKeyTraceReceiverName, _ = tag.NewKey("opencensus_trace_receiver")
+	mReceiverReceivedSpans     = stats.Int64("oc.io/receiver/received_spans", "Counts the number of spans received by the receiver", "1")
+	mReceiverDroppedSpans      = stats.Int64("oc.io/receiver/dropped_spans", "Counts the number of spans dropped by the receiver", "1")
 
-var itemsDistribution = view.Distribution(
-	0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 12, 14, 16, 18, 20, 25, 30, 35, 40, 45, 50, 60, 70, 80, 90,
-	100, 150, 200, 250, 300, 450, 500, 600, 700, 800, 900, 1000, 1200, 1400, 1600, 1800, 2000,
+	tagKeyTraceExporterName, _ = tag.NewKey("opencensus_trace_exporter")
+	mExporterReceivedSpans     = stats.Int64("oc.io/exporter/received_spans", "Counts the number of spans received by the exporter", "1")
+	mExporterDroppedSpans      = stats.Int64("oc.io/exporter/dropped_spans", "Counts the number of spans received by the exporter", "1")
 )
 
-// ViewReceivedSpansReceiver defines the view for the received spans metric.
-var ViewReceivedSpansReceiver = &view.View{
-	Name:        "oc.io/receiver/received_spans",
-	Description: "The number of spans received by the receiver",
-	Measure:     mReceivedSpans,
-	Aggregation: itemsDistribution,
-	TagKeys:     []tag.Key{tagKeyReceiverName},
+// ViewReceiverReceivedSpans defines the view for the receiver received spans metric.
+var ViewReceiverReceivedSpans = &view.View{
+	Name:        mReceiverReceivedSpans.Name(),
+	Description: mReceiverReceivedSpans.Description(),
+	Measure:     mReceiverReceivedSpans,
+	Aggregation: view.Sum(),
+	TagKeys:     []tag.Key{tagKeyTraceReceiverName},
 }
 
-var mExportedSpans = stats.Int64("oc.io/receiver/exported_spans", "Counts the number of exported spans", "1")
+// ViewReceiverDroppedSpans defines the view for the receiver dropped spans metric.
+var ViewReceiverDroppedSpans = &view.View{
+	Name:        mReceiverDroppedSpans.Name(),
+	Description: mReceiverDroppedSpans.Description(),
+	Measure:     mReceiverDroppedSpans,
+	Aggregation: view.Sum(),
+	TagKeys:     []tag.Key{tagKeyTraceReceiverName},
+}
+
+// ViewExporterReceivedSpans defines the view for the exporter received spans metric.
+var ViewExporterReceivedSpans = &view.View{
+	Name:        mExporterReceivedSpans.Name(),
+	Description: mExporterReceivedSpans.Description(),
+	Measure:     mExporterReceivedSpans,
+	Aggregation: view.Sum(),
+	TagKeys:     []tag.Key{tagKeyTraceReceiverName, tagKeyTraceExporterName},
+}
 
-// ViewExportedSpans defines the view for exported spans metric.
-var ViewExportedSpans = &view.View{
-	Name:        "oc.io/receiver/exported_spans",
-	Description: "Tracks the number of exported spans",
-	Measure:     mExportedSpans,
-	Aggregation: itemsDistribution,
-	TagKeys:     []tag.Key{tagKeyExporterName},
+// ViewExporterDroppedSpans defines the view for the exporter dropped spans metric.
+var ViewExporterDroppedSpans = &view.View{
+	Name:        mExporterReceivedSpans.Name(),
```

Comment:
cut & paste error: dropped spans for view fields

Reply:
done.